src/main/resources/application.properties

```diff
@@ -32,6 +32,7 @@ javamelody.management-endpoint-monitoring-enabled=true
 # Health check settings
 health.databaseConnection.timeoutMillis=120000
```

Comment:
I would decrease timeouts to a max of 1 minute.

Reply:
Done